CONTRIBUTING.md

```diff
@@ -29,6 +29,17 @@ To contribute to The Boring JavScript Stack, follow these steps:
 
 ---
 
+## Creating an Issue
+Before **creating** an issue, please follow these steps:
+
+1. Search existing issues before creating a new issue (has someone created an issue already).
+2. If it doesn't exist, create an issue giving as much context and information as possible in the description. 
+> Images and videos can be added to the description to provide more information.
+3. Use an appropriate format for the title:  `[bug]`, `[documentation]`, `[help wanted]`, or `[good first issue]` etc, followed by a concise title description. For example, an issue that relates to The Boring Tech Stack documentation could have a title like `documentation: Add issue section to Contributing guidelines`.
```

Comment:
I think the example didn't reference the guide, so can it be rephrased and also for prefix, let's go with bug:, docs:, help wanted: etc

Hence you example will look like docs: Add issue section to Contributing guidelines

FYI, the angle bracket syntax is intended for PRs

Reply:
I didn't know the angle brackets are for PRs, alright.

I will make the necessary changes and let you know